src/Sylius/Bundle/AdminBundle/Resources/config/routing/admin_user.yml

```diff
@@ -1,16 +1,16 @@
-sylius_admin_admin_user:
-    resource: |
-        alias: sylius.admin_user
-        section: admin
-        path: users
-        templates: "@SyliusAdmin\\shared\\crud"
-        except: ['show']
-        redirect: index
-        grid: sylius_admin_admin_user
-        form:
-            type: Sylius\Bundle\AdminBundle\Form\Type\AdminUserType
-        permission: true
-    type: sylius.resource
+#sylius_admin_admin_user:
+#    resource: |
+#        alias: sylius.admin_user
+#        section: admin
+#        path: users
+#        templates: "@SyliusAdmin\\shared\\crud"
+#        except: ['show']
+#        redirect: index
+#        grid: sylius_admin_admin_user
+#        form:
+#            type: Sylius\Bundle\AdminBundle\Form\Type\AdminUserType
+#        permission: true
+#    type: sylius.resource
```

Comment:
Can we move all route resource to a dedicated file, so we can migrate resources one by one?

Reply:
I think we cannot move routing files. The file is public, so we can break things.
I need to test routing condition to bring a bc-layer with that kind of thing
https://github.com/monsieurbiz/SyliusCmsPagePlugin/blob/8cc79bf5e0a3b66799868ad4ff7e13086f80b347/src/Routing/RequestContext.php#L42